tests/tuple22-tails.scala

```diff
@@ -0,0 +1,4 @@
+class Foo {
+  val tup: (Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int) = ???
+  tup.tail.tail.tail.tail.tail
```

Comment:
Do we want to call `head` to mirror `apply`?

Reply:
Head will become an instance of `apply(0)` therefore the test would be redundant.

Reply:
BTW, do we want to go beyond 22, to test the feature more stressfully :)

Reply:
Will do. Also should put some small tuples.

Reply:
Added